docs/source/models.rst

```diff
@@ -115,37 +207,42 @@ flavors to benefit from all these tools:
 
 Python Function (``python_function``)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The ``python_function`` model flavor serves as a default model interface for MLflow Python models.
+Any MLflow Python model is expected to be loadable as a ``python_function`` model. This enables
+other MLflow tools to work with any python models regardless of which persistence module or
+framework was used to produce the model. This interoperability is very powerful because it allows
+any Python model to be productionized in a variety of environments.
+
+In addition, the ``python_function`` model flavor defines a generic filesystem :ref:`model format
+<pyfunc-filesystem-format>` for Python models and provides utilities for saving and loading models
+to and from this format. The format is self-contained in the sense that it includes all the
+information necessary to load and use a model. Dependencies are stored either directly with the
+model or referenced via conda environment. This model format allows other tools to integrate
+their models with MLflow.
+
+How To Save Model As Python Function
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Most ``python_function`` models are saved as part of other model flavors - for example, all mlflow
+built-in flavors include ``python_function`` flavor in the exported models. In addition, the
+:py:mod:`mlflow.pyfunc` module defines functions for saving ``python_function`` flavor explicitly.
```

Comment:
```suggestion
:py:mod:`mlflow.pyfunc` module defines functions for saving models with the ``python_function`` flavor explicitly.
```

Reply:
Hmm, I am not sure about this one. The module is to save models as python function, there will be no other flavors in this case. "with python function" sounds to me like you can use to store any model that also has python function.

Reply:
Good point. The grammar / wording of the current sentence is a bit confusing, though. How about the following?:

```suggestion
:py:mod:`mlflow.pyfunc` module defines functions for creating ``python_function`` models explicitly.
```